mapstory/templates/maps/map_detail.html

```diff
@@ -29,7 +29,7 @@
   {% autoescape off %}
       $('<iframe>', {
      {% if resource.chapters %}
-     src: '{%  url 'mapstory_view' resource.id %}',
+     src: '{%  url 'mapstory_view' resource.slug %}',
```

Comment:
also, social media share url generation if we want it to use the slugs [here](https://github.com/MapStory/mapstory/blob/87004fa23eab07800f158aaad2dc794737426af4/mapstory/views.py#L1075)